project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java

```diff
@@ -33,6 +33,7 @@
 
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import java.util.Collections;
 
 @WebServlet("/competitionInfo")
 public class CompetitionInfoServlet extends HttpServlet {
```

Comment:
description of what the servlet does

Reply:
I don't know if it can be improved or not